_pages/bea/bea19.md

```diff
@@ -98,32 +96,119 @@ Gold Sponsors
+We ask that all authors read the code, and ensure that their work is conformant to this code.
+Authors are encouraged to devote a section of their paper to concerns about the ethical impact of the work and to a discussion of broader impacts of the work, which will be taken into account in the review process.
+This discussion may extend into a 5th page (short papers) or 9th page (long papers).
+
+Anonymity
+: Given the blind review process, it is essential to ensure that papers remain anonymous.
+Authors should avoid self-references that disclose their identity (e.g., “We previously showed (Smith, 1991)”), opting instead for citations like “Smith previously showed (Smith, 1991)”.
+
+Conflicts of Interest
+: Authors are required to mark potential reviewers who have co-authored the paper, belong to the same research group or institution, or have had prior exposure to the paper, ensuring transparency in the review process.
+
+Double Submissions
+: We adhere to the official ACL double-submission policy.
+If papers are submitted to both BEA and another conference or workshop, authors must specify the other event on the title page (as a footnote on the abstract).
+Additionally, the title page should state that if the paper is accepted for presentation at BEA, it will be withdrawn from other conferences and workshops.
+
+Republications
+: Previously published papers will not be accepted.
+
+### Presentation Guidelines
+
+All accepted papers must be presented at the workshop to appear in the proceedings.
+The workshop will include both in-person and virtual presentation options.
+**At least one author of each accepted paper must register for the conference by the early registration deadline.**
+
+Long and short papers will be presented orally or as posters as determined by the workshop organizers.
+While short papers will be distinguished from long papers in the proceedings, there will be no distinction in the proceedings between papers presented orally and papers presented as posters.
+
+In-person posters
+: Poster stands at the venue will be provided for A0 posters, portrait orientation. This means that your actual poster may be smaller than that but you should make sure it fits on the stand provided. Apart from this, we do not have any specific requirements regarding font types, sizes, etc. (just keep in mind that since you are presenting in person, the poster has to be well readable from a distance, so the font should not be too small).
+
+Online posters
+: We do not have any specific requirements regarding font types, sizes, etc. In line with the physical posters, it might be a good idea to have your digital poster in portrait orientation. During the workshop, you will be presenting your poster in a "virtual" room mimicking the in-person poster sessions (further details to be provided by ACL / Underline). If you have any further technical questions about virtual presentations, please contact Underline directly at acl2023@underline.io.
```

Comment:
Shouldn't [acl2023@underline.io](mailto:acl2023@underline.io) be edited to naacl2024?